kaiax/gov/headergov/impl/init.go

```diff
@@ -82,6 +112,40 @@ func (h *headerGovModule) Init(opts *InitOpts) error {
 
 func (h *headerGovModule) Start() error {
 	logger.Info("HeaderGovModule started")
+
+	lastInsertedBlockPtr := ReadLastInsertedBlock(h.ChainKv)
+	if lastInsertedBlockPtr == nil {
+		// TODO-kaiax: must panic, but commented for unit tests.
+		// panic("last inserted block must exist")
```

Comment:
You can return an error in which case the node will stop at cn.Start()

Reply:
fixed c0e3113